style.css

```diff
@@ -461,6 +451,13 @@ input::placeholder {
   justify-content: center;
 }
 
+.view-cv:hover {
+  background-color: #008552;
+  color: #fff;
+  cursor: pointer;
+  font-size: 1.2em;
+}
+
 .contact-me {
   display: flex;
   flex-direction: column;
```

Comment:
- [ ] Everything is awesome only a small thing is missing which is a small background image as you need to stick to the design as much as possible. Pleas implement it and then you are good to go.
### Missing part
![his](https://github.com/AlyAtalla/AlyAtalla/assets/81727988/faf78a18-f978-40d1-8a40-0d4d065f1b78)